ddclient.in

```diff
@@ -7940,6 +7970,420 @@ sub nic_infomaniak_update {
     }
 }
 
+######################################################################
+## nic_route53_examples
+######################################################################
+sub nic_route53_examples {
+    return <<"EoEXAMPLE";
+
+o 'AWS Route53'
+
+The 'AWS Route53' protocol is used by Route53 service that AWS offers.
+
+The required variables AWS Credentials:
+    - AWS_SECRET_ACCESS_KEY
+    - AWS_ACCESS_KEY_ID
+These will default to your environment variables but can be passed as 
+variables as well.
+
+Configuration variables applicable to the 'route53' protocol are:
+  protocol=route53
+  hosted_zone_id=hosted_zone_id                 ## The id of the Hosted Zone
+  ttl=[number]                                  ## TTL for record (Defaults to 300)
+  region=[aws region]                           ## AWS Region (Defaults to us-east-1)
+  aws-secret-access-key=[AWS_SECRET_ACCESS_KEY] ## Defaults to environment var AWS_SECRET_ACCESS_KEY
+  aws-access-key-id=[AWS_ACCESS_KEY_ID]         ## Defaults to environment var AWS_ACCESS_KEY_ID
+  example.com                                   ## Domain name to update
+
+Example ${program}.conf file entries:
+  protocol=route53,       \\
+  hosted_zone_id=ZXXXXXXXXXXX
+  my.address.com
+
+EoEXAMPLE
+}
+
+sub append_zero {
+    my (
+        $input,
+        $limit
+    ) = @_;
+
+    my $limitValue = $limit // 10;
+    return $input < $limitValue ? '0'.$input : $input;
+}
+
+# Not create a whole package so discount version of a class I guess
+# Start Date object
+sub create_date {
+    my ($time) = @_;
+    my ($sec,$min,$hour,$mday,$mon,$year) = gmtime($time // time);
+
+    my %date = (
+        year => $year + 1900,
+        month => append_zero($mon + 1),
+        day => append_zero($mday),
+        hour => append_zero($hour),
+        min => append_zero($min),
+        sec => append_zero($sec)
```

Comment:
The `append_zero` can be done with `sprintf("%02d\n", $number)` instead

Reply:
Made the change

Reply:
That's a nifty feature, totally forgot about thanks!